agent/acs/handler/payload_handler.go

```diff
@@ -325,6 +325,8 @@ func isTaskStatusNotStopped(status api.TaskStatus) bool {
 // handleUnrecognizedTask handles unrecognized tasks by sending 'stopped' with
 // a suitable reason to the backend
 func (payloadHandler *payloadRequestHandler) handleUnrecognizedTask(task *ecsacs.Task, err error, payload *ecsacs.PayloadMessage) {
+	seelog.Warnf("Received unexpected acs message, messageID: %s, task: %v, err: %v", *payload.MessageId, task, err)
```

Comment:
please use aws.StringValue for printing message id. Also, let's not print the entire task here. Can you just print just the arn instead?

Reply:
I think task arn can be nil here.

Reply:
`aws.StringValue(task.Arn)` should cover that.